pkg/registry/apis/folders/register.go

```diff
@@ -165,14 +168,20 @@ func (b *FolderAPIBuilder) GetAuthorizer() authorizer.Authorizer {
 			}
 
 			action := dashboards.ActionFoldersRead
-			scope := dashboards.ScopeFoldersProvider.GetResourceScopeUID(attr.GetName())
+			scope := dashboards.ScopeFoldersProvider.GetResourceScopeUID(name)
```

Comment:
Nit: I would suggest to rewrite this a bit, something like 
```go
var eval accesscontrol.Evaluator
switch verb {
  case utils.VerbCreate:
    eval = accesscontrol.EvalPermission(dashboards.ActionFoldersCreate)
  case utils.VerbPatch:
    fallthrough
  case utils.VerbUpdate: 
    eval = accesscontrol.EvalPermission(dashboards.ActionFoldersWrite, dashboards.ScopeFoldersProvider.GetResourceScopeUID(name))
  case utils.VerbDeleteCollection:
    fallthrough
  case utils.VerbDelete:
    eval = accesscontrol.EvalPermission(dashboards.ActionFoldersDelete, dashboards.ScopeFoldersProvider.GetResourceScopeUID(name))
}

ok, err := b.accessControl.Evaluate(ctx, user, eval)
if ok {
	return authorizer.DecisionAllow, "", nil
}
eturn authorizer.DecisionDeny, "folder", err
```

Reply:
Ah yes, I see I broke the patch case. Thank you. This is much clearer to read as well.

Reply:
Thanks for the reviews! Looks like there was a default evaluator that I broke when I removed the default action. It's used for "get" for instance. Added it back in here: [5dd4154](https://github.com/grafana/grafana/pull/94514/commits/5dd41540775d4b02f58f68f60f1faa01791a1eac)